src/cc/usdt.h

```diff
@@ -218,12 +219,13 @@ class Context {
   optional<int> pid() const { return pid_; }
   bool loaded() const { return loaded_; }
   size_t num_probes() const { return probes_.size(); }
+  const std::string & cmd_bin_path() { return cmd_bin_path_; }
+  const ino_t inode() { return mount_ns_instance_->target_ino(); }
```

Comment:
Oh. Now I know what you mean, to mark the function as the const, not just return value. That makes sense. Will update.